### PR TITLE
Create a own Enumerable

### PR DESCRIPTION
**Pull request by cmolinan**
### ACTIVITIES

> In this project we learned how to use a module inside a class. For this, we created a class MyList and a module MyEnumerable. 
> The module MyEnumerable implements a subset of the functionality of Enumerable.

### CODE REVIEW

File: src/my_list.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require_relative './enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(*arr)
+    @list = arr
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```
Comment (mikethreels) [inline src/my_list.rb]: - [x] when creating the `each` method the intention is to do this without using the existing `each` method in ruby.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/my_list.rb (hunk 1 above)
Comment (cmolinan) [reply src/my_list.rb]: @mikethreels updated !

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cmolinan** [comment]
@CollinsTatang Thanks !!!

**Pull request merged**